ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/Server.java

```diff
@@ -93,6 +94,14 @@ public Server(String host, int port) {
         this.id = host + ":" + port;
         isAliveFlag = false;
     }
+    
+    public Server(String scheme, String host, int port) {
```

Comment:
Minor nit, can you also update the other constructor to call `this(null, host, port)`?